snap/info.go

```diff
@@ -65,3 +71,13 @@ type AppInfo struct {
 	Plugs   map[string]*PlugInfo
 	Slots   map[string]*SlotInfo
 }
+
+// StoreInfo provides specific information for a store hosted snap.
+type StoreInfo struct {
+	LastUpdated     time.Time
+	DownloadSha512  string
+	DownloadSize    int64
+	AnonDownloadURL string
+	DownloadURL     string
```

Comment:
I suspect that's not true. @mvo5 just fixed a bug last week where things were broken because the non-anonymous one was empty. Why would we use the anonymous one if we have a non-anonymous one? How do we even know which one to use?